src/nav-creator.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-cycle */
+import createMainPage from './main-page';
+import createContactPage from './contact-creator';
+import createMenuPage from './menu-creator';
+/* eslint-enable import/no-cycle */
+
+function createNavigation() {
+  const content = document.getElementById('content');
+
+  const linkMain = '<a href="#" class="main">Go to Main Page</a>';
+  const linkMenu = '<a href="#" class="menu">Go to Menu Page</a>';
+  const linkContact = '<a href="#" class="contact">Go to Contacts Page</a>';
```

Comment:
- [ ] [required] Please use `document.createElement` and `document.appendChild` to create and append HTML elements to the `<div>` with `id=content`. This is an opportunity for you to practice/explore building HTML elements using only javascript. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))